moveit.rosinstall

```diff
@@ -12,14 +12,6 @@
     local-name: moveit_resources
     uri: https://github.com/ros-planning/moveit_resources.git
     version: master
-- git:
```

Comment:
You mention in the commit that this change is temporary. Did you want to eventually restore these?

Reply:
As mentioned in https://github.com/ros-planning/moveit/pull/1288#issuecomment-450533384, after acceptance of this PR, corresponding changes to moveit_visual_tools (any maybe other downstream packages) are needed to cope with the proposed API changes.
Only after these changes are merged too, we can revert `moveit.rosinstall`.

Reply:
> In the maintainer meeting we agreed to keep but deprecate all RobotState::setFromIK functions.

This rosinstall shouldn't need modification since the API isn't changing (yet), please restore